unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -9,11 +9,15 @@
 
 function cert_cleanFiles() {
 
+    if [ -z "$(ls -A ${base_path}/certs)" ]; then
```

Comment:
Wouldn't it be better to use `-d "$DIRECTORY"`? If this IF is to delete the directory, it should be checked if the directory exists, if what you want is to delete its content, then the content should be listed.

Also, if this command is executed, the following ones don't make sense, because the parent directory of those files does not exist, and if this if doesn't enter, the rest of rm won't work because there is no content in cert

if you want to delete all file by file, you should first delete the certificates and at the end the directory with rmdir or directly delete the certs directory regardless of the files it contains

Reply:
Removed in: https://github.com/wazuh/wazuh-packages/commit/eb7df1ce7da929b2267a556c39636f57da812782